.travis-setup.sh

```diff
@@ -17,42 +17,49 @@
 #  along with this program; if not, write to the Free Software
 #  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
-glib_version=2.49.4
+set -e -x
+
+glib_version=2.46.2
```

Comment:
we could do something like that. Actually we need to grep configure.ac for the versions as .travis-setup.sh is run before autogen.sh.

However, we'll also catch when someone bumps the version number but forgets to update .travis-setup.sh, so maybe it can wait a bit longer? I'd like to do it once we've also applied the treatment you've done to glib with json-glib and libcheck (so we can retrieve the version numbers in a single way for all 3 deps).